src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const unique = Array.from(new Set(values));
+
+  if (values.length !== unique.length) {
+    return null;
+  }
+
+  const res = {};
```

Comment:
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)